simulator/test_remote.py

```diff
@@ -1,17 +1,26 @@
 import time
+import signal
+import sys
 
 from poppy.creatures import PoppyErgoJr
 from threading import Thread
 
 jr = PoppyErgoJr(simulator='threejs', use_http=True)
 
-# Magie Python pour lancer le server web en background
-
 t = Thread(target=jr.http.run)
 t.daemon = True
 t.start()
 
+print('PoppyErgoJr simulation started')
+
 jr.dance.start()
 
-while True:
-    time.sleep(1000)
+
+def signal_handler(signal, frame):
+    print('\nExiting viewer')
+    sys.exit(0)
+
+
+signal.signal(signal.SIGINT, signal_handler)
+print('Press Ctrl+c to quit')
+signal.pause()
```

Comment:
Out of curiosity, this does not have the exact same behaviour than the previous version?